rd-net/RdFramework.Reflection/RdFramework.Reflection.csproj

```diff
@@ -0,0 +1,71 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;net35;net461</TargetFrameworks>
+        <AssemblyName>JetBrains.RdFramework.Reflection</AssemblyName>
+        <RootNamespace>JetBrains.Rd.Reflection</RootNamespace>
+
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <NoWarn>1591,1570,1574</NoWarn>
+
+        <DefineConstants Condition="'$(Configuration)' == 'Debug'">JET_MODE_ASSERT</DefineConstants>
+
+        <Configurations>Debug;Release;CrossTests</Configurations>
+
+        <Platforms>AnyCPU</Platforms>
+    </PropertyGroup>
+
+
+    <PropertyGroup Label="NuGet">
+        <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+        <PackageId>JetBrains.RdFramework.Reflection</PackageId>
+        <PackageProjectUrl>https://github.com/JetBrains/rd</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/JetBrains/rd</RepositoryUrl>
+        <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+        <RepositoryType>git</RepositoryType>
+        <Authors>JetBrains</Authors>
+        <Copyright>Copyright © JetBrains 2019</Copyright>
+        <PackageTags>rd reactive distributed protocol</PackageTags>
+        <Description>Plugin for JetBrains Networking library (JetBrains.RdFramework) used for defining models using regular C# for C#-C# communications</Description>
```

Comment:
Looks like copy-paste. It will be visible on NuGet.org.

Reply:
You pointed out to description tag. What exactly wrong with it? May you suggest a better description?